eppo_client/assignment_logger.py

```diff
@@ -1,6 +1,10 @@
 from typing import Dict
+from eppo_client.base_model import BaseModel
+from pydantic import ConfigDict
 
 
-class AssignmentLogger:
+class AssignmentLogger(BaseModel):
+    model_config = ConfigDict(arbitrary_types_allowed=True)
```

Comment:
`AssignmentLogger` does not inherit from `SdkBaseModel` because it has unique serialization requirements from the json dto objects